src/pages/IndexPage.vue

```diff
@@ -1,16 +1,48 @@
 <template>
-  <q-page class="flex flex-center">
-    <h1>
-      <em class="fa-solid fa-person-digging"></em>
-      {{ $t('project.name') }}
-    </h1>
-  </q-page>
+  <div class="flex flex-center main">
+    <div class="fit row justify-center items-center">
+      <q-img
+        src="https://placeimg.com/500/300/nature"
+        style="height: 140px; max-width: 150px"
+      />
+    </div>
+  </div>
+  <div class="flex flex-center">
+    <q-btn
+      color="primary"
+      size="xl"
+      label="Create empty project"
+      :to="`/modelizer/${projectId}/model`"
+    />
+  </div>
 </template>
 
 <script>
 import { defineComponent } from 'vue';
 
+/**
+ * Makes an hexadecimal string of the specified length
+ * @param {number} length
+ */
+function randomHexString(length) {
+  const buffer = new Uint32Array(length);
+  window.crypto.getRandomValues(buffer);
+  return Array.from(buffer, (i) => (i % 16).toString(16)).join('');
+}
+
 export default defineComponent({
   name: 'IndexPage',
+  setup() {
+    return {
+      projectId: randomHexString(8),
```

Comment:
Prefere to have a `composables/Project.js` to have all function of the project like create, delete etc etc.